main.js

```diff
@@ -3,9 +3,36 @@ const EMPTY_HEART = '♡'
 const FULL_HEART = '♥'
 
 // Your JavaScript code goes here!
+const handleServerError = (err) => {
+  const modal = document.querySelector('#modal')
+  modal.querySelector('#modal-message').textContent = err
+  modal.className = ''
+  setTimeout(hideErrorModal,3000)
```

Comment:
There should always be spacing after commas. 
```suggestion
  setTimeout(hideErrorModal, 3000)
```